package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "vscode-camelk",
-	"displayName": "Tooling for Apache Camel-K",
-	"description": "VS Code extensions to support Camel-K functionality",
+	"displayName": "Tooling for Apache Camel K",
+	"description": "VS Code extensions to support Apache Camel K functionality",
```

Comment:
extensions? Its just this single one, isn't it?

Reply:
Will change to "VS Code support for Apache Camel K functionality"